src/data/account/__test__/account.test.js

```diff
@@ -4,6 +4,7 @@ import {
   validateResetPasswordTokenAsync,
```

Comment:
Other unit tests are failing now: @vdanylov 

  ● Test for reset password token validation › reject with statusCode 404 if the reset password is invalid

    Error
      [object Object] thrown

Reply:
@maartenhemmes strange I fire this test and it passes.

Reply:
@maartenhemmes I think You have got old version, cause I resolved these conflicts before. I'll check it one more time.

Reply:
ok. on dev it's working fine, in this PR it's failing:

<img width="910" alt="screen shot 2018-04-21 at 7 24 18 am" src="https://user-images.githubusercontent.com/3624039/39077515-0eb000b4-4535-11e8-9ace-aa8d55d28e6a.png">
![Uploading Screen Shot 2018-04-21 at 7.23.31 AM.png…]()

Reply:
<img width="877" alt="screen shot 2018-04-21 at 7 23 31 am" src="https://user-images.githubusercontent.com/3624039/39077528-2253c466-4535-11e8-8dee-673647c6bf30.png">

Reply:
@maartenhemmes thanks, I debugged problem and fix it.